Wikia/AlphaB
# Add MVP version of the library

## Goal of the PR

Create an initial version of the library that handles A/B testing (two groups in the same data frame only) by generating charts and computing statistical significance considering a certain variable.

## How to test it

* Run `pip install -e git+http://github.com/Wikia/AlphaB@mvp-library#egg=AlphaB` (this command will create folders/files in your current working directory, so it's recommended to run it in `/tmp` for example)
* Clone the repository and checkout the branch `mvp-library`
* Run `pip install -r requirements.txt`
* Run `jupyter notebook` and select the notebook in `example/example.ipynb`
    * Run all of the cells in the notebook, it should work and both the library and the manual approach should yield the exact same results
* If you like, play around with the `render` method options and by using different datasets than the provided example one
